toolkit/tools/scheduler/schedulerutils/printresults.go

```diff
@@ -334,3 +335,44 @@ func unbuiltPackagesCSVRows(pkgGraph *pkggraph.PkgGraph, unbuiltPackages, failed
 
 	return
 }
+
+// Helper function to print summarized numbers of the build to the logger.
+func printSummary(failedSRPMs, failedSRPMsTests map[string]*BuildResult, prebuiltSRPMs, prebuiltDeltaSRPMs, builtSRPMs, testedSRPMs, skippedSRPMsTests, unresolvedDependencies map[string]bool, blockedSRPMs, blockedSRPMsTests map[string]*pkggraph.PkgNode, rpmConflicts, srpmConflicts []string, allowToolchainRebuilds bool, conflictsLogger func(format string, args ...interface{})) {
+	logger.Log.Info("---------------------------")
+	logger.Log.Info("--------- Summary ---------")
+	logger.Log.Info("---------------------------")
+
+	logger.Log.Infof(color.GreenString("Number of prebuilt SRPMs:           " + fmt.Sprint(len(prebuiltSRPMs))))
```

Comment:
Can we use the previous string formatting? It looks like `GreenString` (and other colours) should already support something like this:
```suggestion
	logger.Log.Infof(color.GreenString("Number of prebuilt SRPMs:           %d", (len(prebuiltSRPMs)))
```